common/persistence/dataInterfaces.go

```diff
@@ -1196,6 +1223,26 @@ func UnixMilliseconds(t time.Time) int64 {
 	return unixNano / int64(time.Millisecond)
 }
 
+func ParseHistoryBranchToken(branchToken []byte) (*persistencespb.HistoryBranch, error) {
+	return serialization.HistoryBranchFromBlob(branchToken, enumspb.ENCODING_TYPE_PROTO3.String())
+}
```

Comment:
Those two functions can be unified? I don't see any call to serialization.HistoryBranchFromBlob using an encoding different than PROTO3.

Reply:
Is this resolved?